src/backend/utils/adt/dbsize.c

```diff
@@ -489,7 +489,7 @@ pg_relation_size_oid(PG_FUNCTION_ARGS)
 
 		initStringInfo(&buffer);
 
-		appendStringInfo(&buffer, "select sum(pg_relation_size('%s.%s'))::int8 from gp_dist_random('gp_id');", quote_identifier(schemaName), quote_identifier(relName));
+		appendStringInfo(&buffer, "select sum(pg_relation_size(%u))::int8 from gp_dist_random('gp_id');", relOid);
```

Comment:
Do we have guarantees that the table oid consist across the cluster?
Inconsistent oids sound like madness, but code before your change looks like to try to handle this case.

Reply:
I suppose it is a kind of vestige.
[This](https://github.com/greenplum-db/gpdb/commit/1843ec62be5a36c573c4706da0aa2a3e9cd599e3#diff-05d3a0c9a7085cced205f8b5e7820fa77f0cc62e43c416482a28d10ecd60caebR44) commit seems to address the similar escaping problem. There used to be uncertainty, whether all OIDs are synced across the cluster. However now consistency is stated (commit is included into 5X).